addons/nuxeo-csv/nuxeo-csv-core/src/test/java/org/nuxeo/ecm/csv/core/TestCSVImporterCreateMode.java

```diff
@@ -89,6 +89,8 @@
     private static final String DOCS_WITH_LEGACY_DATE_FORMAT_CSV = "docs_with_legacy_date_format.csv";
 
     private static final String DOCS_WITH_CUSTOM_DATE_FORMAT_CSV = "docs_with_custom_date_format.csv";
+    public static final String FILE_GREATER_THAN_THE_LENGTH_THRESHOLD = "file_greater_than_threshold.csv";
+    public static final String FILE_BELOW_THE_LENGTH_THRESHOLD = "file_below_the_threshold.csv";
```

Comment:
Please add blank lines between the constants.

Reply:
and i will add `@since` on these constantes to be consistent